src/components/SwitchActiveAffiliation/SwitchActiveAffiliation.js

```diff
@@ -26,6 +26,7 @@ export const SwitchActiveAffiliation = ({ stripes }) => {
   const history = useHistory();
   const modules = useModules();
   const [open, toggle] = useToggle(true);
+  const [submitting, toggleSubmit] = useToggle(false);
```

Comment:
This could be improved. `[isSubmitting, setIsSubmitting]`

Reply:
it's not set, it's toggle, the same for submitting, we follow the same pattern for useToggle hook